__tests__/app.test.js

```diff
@@ -32,11 +32,43 @@ describe('/api', () => {
         .get('/api')
         .expect(200)
         .then(({body}) => {
-            return fs.readFile('./endpoints.json')
-            .then((endpoints) => {
-                expect(body.endpoints).toEqual(JSON.parse(endpoints))
-            })
-            
+            expect(body.endpoints).toEqual(endpoints)
+        })
+    })
+});
+describe('/api/articles/:article_id', () => {
+    test('GET: responds with an article object: author, title, article_id, body, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+        .get('/api/articles/3')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual({
+                article_id: 3,
+                title: 'Eight pug gifs that remind me of mitch',
+                topic: 'mitch',
+                author: 'icellusedkars',
+                body: 'some gifs',
+                created_at: '2020-11-03T09:12:00.000Z',
+                votes: 0,
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+              })
+
+        })
+    });
+    test('GET: if passed an invalid id number, responds with an empty object', () => {

```

Comment:
This shouldn't be a 200. If the article id currently doesn't exist _but_ could possibly exist in the future, what should happen here instead?